packages/arbos-contracts/package.json

```diff
@@ -14,18 +14,23 @@
   },
   "homepage": "https://offchainlabs.com/",
   "scripts": {
-    "build": "truffle compile"
+    "build": "hardhat compile",
+    "typechain": "hardhat typechain"
```

Comment:
I'm assuming we need this to create the typechain for the arb-os precompiles. Can't we access those through the git submodule? If a regular import doesn't work I've got a few hacky fun ideas - which might be better than needing to keep these two in sync.

Reply:
And if we do keep this workspace, we could rename it to `arb-os-contracts`?

Reply:
Ah yeah, this should really be in arb-os. Would probably make sense to make a PR there to get that setup